topics/dev/tutorials/tool-from-scratch/tutorial.md

```diff
@@ -147,8 +158,8 @@ Recipes should always define the following 6 sections in the `meta.yaml` file:
 >
 >     {% endraw %}
 >
-> 5. Next, we move on to the build metadata. Since this is the first version of the conda recipe, the build number is 0.
-> We use the externally defined `{{ PYTHON }}` variable, which defines which python conda-build is using, to install it
+> 8. Next, we move on to the build metadata. Since this is the first version of the conda recipe, the build number is 0.
+> We use the externally defined `{% raw %}{{ PYTHON }}{% endraw %}` variable, which defines which python conda-build is using, to install it
```

Comment:
:fearful:

Reply:
Yep, it was not displayed, not sure if there's a simpler syntax trick?

Reply:
No, that's right, I'm just horrified we missed it

Reply:
Yeah it's easy to miss this kind of problem when reviewing

Reply:
https://jekyllrb.com/docs/configuration/liquid/ looks like we might be able to set a stricter liquid setting and catch those. I'll test it out (unrelated to the PR)